identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -84,6 +86,35 @@ trait ConsentsJourney
       }
     }
 
+  private def displayConsentComplete(
+    page: ConsentJourneyPage,
+    consentHint: Option[String]): Action[AnyContent] =
+    csrfAddToken {
+      consentsRedirectAction.async { implicit request =>
```

Comment:
Hmm worth checking the behaviour for an unvalidated email address hitting this endpoint.  Have we thought about what we want the behaviour to be?

Reply:
Looks like we can still access this page with an unverified email, needs to be fixed in another PR.